roles/analytics-dashboard/tasks/main.yml

```diff
@@ -58,17 +69,32 @@
         remote_src: yes
       when: pack_location == "remote"
 
-#    - name: Copy JDBC Driver
-#      copy:
-#       src: "{{ product_package_location }}/lib/{{ jdbc_driver }}"
-#       dest: "{{ carbon_home }}/repository/components/lib"
+    - name: Copy JDBC Driver
+      copy:
+       src: "{{ product_package_location }}/lib/{{ jdbc_driver }}"
+       dest: "{{ carbon_home }}/wso2/analytics/lib"
 
     - name: Copy configuration templates
       template:
         src: "{{ item.src }}"
         dest: "{{ item.dest }}"
       loop: "{{ config_files }}"
 
+# Enable customization by uncommenting the following and adding the changes to the custom.yml
+    - import_tasks: custom.yml
+
+    - name: Run jar to bundle script
+      command: "sh {{ carbon_home }}/wso2/analytics/bin/jartobundle.sh {{ carbon_home
+      }}/wso2/analytics/lib/{{jdbc_driver }} {{ carbon_home }}/wso2/analytics/lib"
+      environment:
+        JAVA_HOME: "{{java_home}}"
+      when: jdbc_driver is defined
+
+    - name: Remove default database connector jar file
+      file:
+        path: "{{ carbon_home }}/wso2/analytics/lib/{{jdbc_driver}}"
+        state: absent
+
```

Comment:
Do not have custom scripts in the GitHub project.

Reply:
fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243